### PR TITLE
Refresh me page gravatar once the email for the account is updated

### PR DESCRIPTION
**Pull request by oguzkocer**
We trigger an update on the email and default blog when the Me page loads for the first time, updating the email will now trigger an update on the gravatar. This is especially important for the initial update the 4.8 version, since we are starting to store the email in this version. The gravatar is also changed to be round.

@aerych May I bother you with another review?

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Me/MeHeaderView.m [hunk 1]
```diff
@@ -104,6 +104,8 @@ - (UIImageView *)imageViewForGravatar
 {
     CGRect gravatarFrame = CGRectMake(0.0f, 0.0f, MeHeaderViewGravatarSize, MeHeaderViewGravatarSize);
     UIImageView *imageView = [[UIImageView alloc] initWithFrame:gravatarFrame];
+    imageView.layer.cornerRadius = MeHeaderViewGravatarSize / 2.0;
+    imageView.clipsToBounds = YES;
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/Me/MeHeaderView.m]: Yay round avatar :D

File: WordPress/Classes/Services/AccountService.h [hunk 2]
```diff
@@ -4,6 +4,7 @@
 @class WPAccount, Blog;
 
 extern NSString *const WPAccountDefaultWordPressComAccountChangedNotification;
+extern NSString *const WPAccountWordPressComAccountEmailAndDefaultBlogUpdatedNotification;
```
Comment (aerych) [inline WordPress/Classes/Services/AccountService.h]: Zomg, 66 chars. Could this const be smaller? Please? Pretty please with cherry on top?

**Comment by aerych** [comment]
Looks great Oguz! Loving that the avatar is rounded now. On small nitpicky comment about a const's name but otherwise :shipit: when ready!

File: WordPress/Classes/Services/AccountService.h (hunk 2 above)
Comment (oguzkocer) [inline WordPress/Classes/Services/AccountService.h]: Changed it to `WPAccountEmailAndDefaultBlogUpdatedNotification`.

**Comment by oguzkocer** [comment]
Thanks @aerych! I've updated the notification name, I was just trying to be consistent with the previous notification name there :)

**Pull request merged**